FrontEndApp/autodl-platform/.env.example

```diff
@@ -0,0 +1,8 @@
+# server url
+NEXT_APP_BACKEND_API_URL=http://localhost:8000
+
+# recommendation system url
+NEXT_APP_FASTAPI_URL=http://localhost:8080
+
+# upload to github
+NEXT_APP_GITHUB_APP_CLIENT_ID=abcdoadsnfioasndf343
```

Comment:
For NextJS it's `NEXT_PUBLIC_` but doing so will make the variable also accessible in the browser. So we don't need any prefixes.
https://nextjs.org/docs/basic-features/environment-variables

Reply:
Yes I would've removed the prefixes, but felt like keeping it similar to the React one. 😅
I'll remove it right away ~